src/main/java/org/springframework/data/web/ProxyingHandlerMethodArgumentResolver.java

```diff
@@ -80,28 +83,36 @@ public void setBeanClassLoader(ClassLoader classLoader) {
 	@Override
 	public boolean supportsParameter(MethodParameter parameter) {
 
+		// Simple type or not annotated with @ModelAttribute (and flag set to require annotation)
 		if (!super.supportsParameter(parameter)) {
 			return false;
 		}
 
 		Class<?> type = parameter.getParameterType();
 
+		// Only interfaces can be proxied
 		if (!type.isInterface()) {
 			return false;
 		}
 
-		// Annotated parameter (excluding multipart)
-		if ((parameter.hasParameterAnnotation(ProjectedPayload.class) || parameter.hasParameterAnnotation(
-				ModelAttribute.class)) && !MultipartResolutionDelegate.isMultipartArgument(parameter)) {
+		// Multipart not currently supported
```

Comment:
I reworked this a bit to just pull the fact that this proxy resolver does not handle multipart, regardless of what annotations it does or does not have. This may be adjusted as part of #3299